sdk/communication/azure-communication-identity/src/main/java/com/azure/communication/identity/util/CommunicationIdentityClientUtils.java

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.communication.identity.util;
+
+import com.azure.communication.identity.CommunicationIdentityAsyncClient;
+import com.azure.communication.identity.CommunicationIdentityClient;
+import com.azure.communication.identity.implementation.models.CommunicationIdentityAccessTokenRequest;
+import com.azure.communication.identity.implementation.models.CommunicationIdentityCreateRequest;
+import com.azure.communication.identity.models.CommunicationTokenScope;
+import com.azure.core.util.logging.ClientLogger;
+
+import java.time.Clock;
+import java.time.Duration;
+import java.time.OffsetDateTime;
+import java.time.temporal.ChronoUnit;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+
+/**
+ * Utility class with shared functionality for {@link CommunicationIdentityClient}, {@link CommunicationIdentityAsyncClient}
+ * and their test classes.
+ */
+public final class CommunicationIdentityClientUtils {
```

Comment:
remove pls public access modifier, it should not be part of public API

Reply:
Made package private and moved to the package with clients and tests.